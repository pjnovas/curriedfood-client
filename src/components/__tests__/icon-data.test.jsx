import IconData from '../icon-data.jsx';

jest.mock('../icon');

const props = {
  icon: 'clock',
  text: '15"'
};

takeSnapshots(
  {
    default: {
      ...props
    },
    reversed: {
      ...props,
      reverse: true
    },
    asChildren: {
      ...props,
      children: 'BOOM!'
    }
  },
  IconData
);
