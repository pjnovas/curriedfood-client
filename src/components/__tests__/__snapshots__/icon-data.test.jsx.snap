// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asChildren 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      },
      Object {},
    ]
  }
>
  <Icon
    color="rgba(255, 255, 255, 0.38)"
    name="clock"
    size={20}
    style={
      Object {
        "marginHorizontal": 5,
      }
    }
  />
  BOOM!
</View>
`;

exports[`default 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      },
      Object {},
    ]
  }
>
  <Icon
    color="rgba(255, 255, 255, 0.38)"
    name="clock"
    size={20}
    style={
      Object {
        "marginHorizontal": 5,
      }
    }
  />
  <withTheme(Text)
    style={
      Object {
        "color": "rgba(255, 255, 255, 0.38)",
      }
    }
  >
    15"
  </withTheme(Text)>
</View>
`;

exports[`reversed 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      },
      Object {
        "flexDirection": "row-reverse",
      },
    ]
  }
>
  <Icon
    color="rgba(255, 255, 255, 0.38)"
    name="clock"
    size={20}
    style={
      Object {
        "marginHorizontal": 5,
      }
    }
  />
  <withTheme(Text)
    style={
      Object {
        "color": "rgba(255, 255, 255, 0.38)",
      }
    }
  >
    15"
  </withTheme(Text)>
</View>
`;
